app/assets/stylesheets/pages/mentor-configure.scss

```diff
@@ -39,12 +43,16 @@
 
     label {
         background:rgba($color-2, 0.1);
+        border:1px solid rgba($color-2, 0.2);
```

Comment:
No variable color?

Reply:
This is what happens when I make things up rather than going via the official channel of Nicole :) I don't think I have the commit-bit to define new colours ;)